### PR TITLE
fix: revert pfm from 7.1.2 to 7.1.1

### PR DESCRIPTION
**Pull request by czarcas7ic**
Closes: #XXX

## What is the purpose of the change

E2E is not compiling from this upgrade. It works locally, but I don't think the overhead of understanding what is going on is worth the time, so just reverting it.

### CODE REVIEW

**Review by PaddyMc — APPROVED** [review]

**Review by nicolaslara — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -28,7 +28,7 @@ require (
 	github.com/osmosis-labs/go-mutesting v0.0.0-20221208041716-b43bcd97b3b3
 	github.com/osmosis-labs/osmosis/osmomath v0.0.7-0.20231124190325-d75e9ade352e
 	github.com/osmosis-labs/osmosis/osmoutils v0.0.7-0.20231124190325-d75e9ade352e
-	github.com/osmosis-labs/osmosis/x/epochs v0.0.3-0.20231124190325-d75e9ade352e
+	github.com/osmosis-labs/osmosis/x/epochs v0.0.3-0.20231128160617-9a7cef4bff58
```
Comment (nicolaslara) [inline go.mod:31]: this seems unrelated

Comment (czarcas7ic) [reply go.mod:31]: I just called the action that updates all the go mods
